src/components/MenuItems.tsx

```diff
@@ -211,49 +211,59 @@ const MenuItems: React.FC<MenuItemsProps> = ({ currentRole, callsData }) => {
 
   const user = (
     <div data-cy="user-menu-items">
-      <ListItem component={NavLink} to="/" exact button>
```

Comment:
**For Reviewer:**
GitHubs preview of these changes seems to be confused.
Here is **a better preview of the changes** for this file:
![image](https://user-images.githubusercontent.com/31534888/117114830-39168e80-ad84-11eb-93b7-4a1cf1c8026a.png)